### PR TITLE
Partly revert #12337

### PR DESCRIPTION
**Pull request by HLeithner**
Logout redirect returns to the default page not to the same page.

Pull Request for Issue #25941 .

### Summary of Changes
Reverted language string to reflect behavior.

### Testing Instructions
Create a menu item with com_users logout view.

### Expected result
Correct 'Logout Redirection Page' description

### Actual result
Description describes a wrong behavior for the default value.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -540,7 +540,7 @@ JGLOBAL_SEF_ADVANCED_MODERN="Modern"
 JGLOBAL_SEF_NOIDS_DESC="Remove the IDs from the URLs of this component."
 JGLOBAL_SEF_NOIDS_LABEL="Remove IDs from URLs"
 JGLOBAL_SEF_TITLE="Routing"
-JGLOBAL_SELECT_ALLOW_DENY_GROUP="Change %s permission for %s group."
+JGLOBAL_SELECT_ALLOW_DENY_GROUP="Change %s permission for %s group.""
```
Comment (SharkyKZ) [inline administrator/language/en-GB/en-GB.ini]:
```suggestion
JGLOBAL_SELECT_ALLOW_DENY_GROUP="Change %s permission for %s group."
```

**Comment by infograf768** [comment]
@HLeithner 
Don't we have a similar issue with login?

**Comment by simbus82** [comment]
For me this PR is wrong.
https://github.com/joomla/joomla-cms/issues/25941#issuecomment-523017228

**Comment by HLeithner** [comment]
Maybe I'm wrong I'm closing this.

**Pull request closed without merging**